src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -24461,7 +24501,7 @@ static void F<T>()
             var declarators = tree.GetRoot().DescendantNodes().OfType<VariableDeclaratorSyntax>().ToArray();
             var symbol = (LocalSymbol)model.GetDeclaredSymbol(declarators[0]);
             Assert.Equal("T", symbol.Type.ToTestDisplayString(true));
-            Assert.Equal(true, symbol.Type.IsNullable);
+            Assert.False(symbol.Type.IsNullable);
```

Comment:
>Assert.False(symbol.Type.IsNullable); [](start = 12, length = 37)

Is this the behavior we expect for public API as well, when asking about unconstrained type parameter? 
It feels like the public and internal APIs should probably align for this question.

Reply:
> Is this the behavior we expect for public API as well, when asking about unconstrained type parameter? 
It feels like the public and internal APIs should probably align for this question.

We are still figuring out whet the shape of public API should be.
I believe this internal API is misleading and I am planning to get rid of it internally.

---
In reply to: [230538988](https://github.com/dotnet/roslyn/pull/30913#discussion_r230538988) [](ancestors = 230538988)